agent/containers/images/Makefile

```diff
@@ -29,7 +29,7 @@ _REPO_TEMPLATE = ../../ansible/pbench/agent/roles/pbench_repo_install/templates/
 
 # The list of RPMs which provide the various tools we offer.
 # Not intended to be overridden with an environment variable.
-_TOOL_RPMS = pbench-sysstat perf blktrace bpftrace numactl pcp-system-tools strace kernel-tools tcpdump
+_TOOL_RPMS = atomic-openshift-clients blktrace bpftrace cpupowerutils golang kernel-tools libvirt-client nmap-ncat numactl pbench-sysstat pcp-system-tools perf procps-ng strace tcpdump trace-cmd
```

Comment:
So what happened here?

Reply:
The long of the short of it is that the list was incomplete, and when @ndokos pointed out that we needed to remove `procps-ng` from the agent RPM `.spec` file, I checked the `agent/tool-scripts/base-tool` file to see what packages were recorded there for tools.  I just took that list, bumped it against the existing, and presented them in sorted order.